File: unificacion_maga/Databases/extract_inserts.py
import re

with open('Databases/dump-EjecucionPresupuestaria-202605251318.sql', 'r', encoding='utf-8') as f:
    for line in f:
        if line.startswith('INSERT INTO'):
            match = re.match(r'INSERT INTO `([^`]+)`', line)
            if match:
                print("Found INSERT for table:", match.group(1))

File: unificacion_maga/Databases/extract_inserts.php
<?php
$file = file_get_contents('c:\\xampp\\htdocs\\unificacion_maga\\Databases\\dump-EjecucionPresupuestaria-202605251318.sql');
preg_match_all("/INSERT INTO `([^`]+)`/i", $file, $matches);
$tables = array_unique($matches[1]);
foreach($tables as $table) {
    echo "Found INSERT for table: $table\n";
}
